src/Ztm.WebApi/InvalidCallbackUrlException.cs

```diff
@@ -1,24 +1,20 @@
+using System;
 using System.Net;
 
 namespace Ztm.WebApi
 {
     public sealed class InvalidCallbackUrlException : HttpResponseException
     {
-        public InvalidCallbackUrlException(string InvalidUrl)
+        static readonly string InvalidCallbackUrlTitle = "Invalid Callback Url";
```

Comment:
Rename to just `StaticTitle` looks better due to we already in `InvalidCallbackUrlException` class.